job_runner/jobs/email_scan/indeed_utils/index.test.js

```diff
@@ -3,7 +3,7 @@ import { scanIndeedEmail } from '.';
 
 const { cwd } = process;
 
-const samplePath = `${cwd()}/job_runner/jobs/email_scan/indeed_utils/sample_indeed_email.html`;
+const samplePath = `${cwd()}/jobs/email_scan/indeed_utils/sample_indeed_email.html`;
```

Comment:
I fixed this in my `cicd` branch/PR #16 by using __dirname which I think is safer.
Can you try running that branch? as part of the making cicd work, I got made sure all the tests were working.